app/src/main/java/xyz/malkki/neostumbler/utils/tileJson.kt

```diff
@@ -0,0 +1,36 @@
+package xyz.malkki.neostumbler.utils
+
+import android.util.Log
+import java.io.IOException
+import okhttp3.Call
+import okhttp3.Callback
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.Response
+import org.json.JSONObject
+
+fun getTileJsonLayerIds(tileJson: String, callback: (List<String>) -> Unit) {
+    val layerIds = mutableListOf<String>()
+
+    OkHttpClient().newCall(Request.Builder().url(tileJson).build()).enqueue(object : Callback {
+        override fun onFailure(call: Call, e: IOException) {
+            Log.e("Network", "Request failed: ${e.message}")
```

Comment:
Use Timber for logging

Reply:
Done via https://github.com/mjaakko/NeoStumbler/compare/93d572b5d4ec354df5c1113b07c5300d79f21904..6a973c91edb0b80ec4b15eb26ec93b293984c403